.github/BOTMETA.yml

```
@@ -679,7 +679,7 @@ files:
   $modules/jenkins_script.py:
     maintainers: hogarthj
   $modules/jira.py:
-    ignore: DWSR
+    ignore: DWSR tarka
     labels: jira
     maintainers: Slezhuk tarka pertoft
```

You also need to remove yourself here:
```suggestion
    maintainers: Slezhuk pertoft
```

---

Thanks @felixfontein. PR updated.